Session/Utilities/UIContextualAction+Utilities.swift

```diff
@@ -231,6 +231,32 @@ public extension UIContextualAction {
                             indexPath: indexPath,
                             tableView: tableView
                         ) { _, _, completionHandler in
+                            let isPinning: Bool = (threadViewModel.threadPinnedPriority <= 0)
```

Comment:
In this case I think `isPinning` is fine, but we have a bunch of other explicit `threadViewModel.threadPinnedPriority > 0` checks in this switch case so I wonder if this variable should be created at the start of the case and the code updated to use `isPinning` in all cases instead of the `threadViewModel.threadPinnedPriority` comparison?